build.cmd

```diff
@@ -15,6 +18,13 @@ set unprocessedBuildArgs=
 :Loop
 if [%1]==[] goto Tools
 
+if /i "%1" == "/?"    goto Usage
+if /i "%1" == "-?"    goto Usage
+if /i "%1" == "/h"    goto Usage
+if /i "%1" == "-h"    goto Usage
+if /i "%1" == "/help" goto Usage
+if /i "%1" == "-help" goto Usage
```

Comment:
Maybe you could have all of the help options contained in one variable, so you could do:

```cmd
set "HelpOptions=-? /? -h /h help /help -help --help"

for %%o in (%HelpOptions%) do (
    if /i "%~1" == "%%o" goto Usage
)
```

Also note that you should probably use "%~1" instead of "%1", since the former strips the quotes from the first argument. So i.e. if you run `build "-?"`, `"%~1"` evaluates to `"-?"` while `"%1"` is `""-?""`, which cmd chokes on.